src/corehost/cli/args.cpp

```diff
@@ -89,7 +89,8 @@ bool parse_arguments(const int argc, const pal::char_t* argv[], arguments_t& arg
             {
                 trace::error(_X("Failed to locate deps file: %s"), args.deps_path.c_str());
                 return false;
-            }            
+            }      
+            args.app_dir = get_directory(args.deps_path);      
             args.app_argc = args.app_argc - 1;
```

Comment:
@schellap this is required for the XUnit test scenario where the entrypoint EXE is not in the actual app-base. However, the invariant (which we already hold) is that app-base is always where the deps file is and for the default/common cases, that is where the entrypoint EXE lives.

Ofcourse, with this change, the assumption is that the entrypoint EXE has all of its dependencies either in appbase or in the deps file.